cdc/entry/mounter.go

```diff
@@ -442,6 +442,7 @@ func (m *mounterImpl) mountRowKVEntry(tableInfo *model.TableInfo, row *rowKVEntr
 		Table: &model.TableName{
 			Schema:    schemaName,
 			Table:     tableName,
+			TableID:   row.PhysicalTableID,
 			Partition: partitionID,
```

Comment:
we can remove the `Partition` now

Reply:
How about remove the Partition in another PR?

Reply:
ok